projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -6,20 +6,58 @@ import {
 import { Spinner, BoostScoreBar } from '@automattic/jetpack-components';
 import { Popover } from '@wordpress/components';
 import { useViewportMatch } from '@wordpress/compose';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { arrowUp, Icon } from '@wordpress/icons';
 import { useEffect, useState, useMemo, useCallback } from 'react';
 import { PRODUCT_STATUSES } from '../../../constants';
 import useProduct from '../../../data/products/use-product';
 import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
 import useAnalytics from '../../../hooks/use-analytics';
 import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
+import formatPercentage from '../../../utils/format-percentage';
 import { useBoostTooltipCopy } from './use-boost-tooltip-copy';
 import type { SpeedScores, BoostSpeedScoreType } from './types';
 import type { SetStateAction } from 'react';
 
 import './style.scss';
 
+const createSpeedScoreSRText = (
+	speedLetterGrade: string,
+	currentSpeedScore: number,
+	boostScoreIncrease?: number,
+	cta?: string
+) => {
+	const fragments = [];
+
+	fragments.push(
+		sprintf(
+			// translators: %1$s: speed grade (e.g. 'A'), %2$s: speed score percentage (e.g. '95%').
+			__( 'Your website’s overall speed score is %1$s, or %2$s.', 'jetpack-my-jetpack' ),
+			speedLetterGrade,
+			formatPercentage( currentSpeedScore / 100 )
```

Comment:
@Automattic/heart-of-gold, is it okay to refer to the Boost score as a percentage? Or should I stick to a plain number? I'm just concerned the latter wouldn't make much sense without a scale.

Reply:
Hey @monsieur-z!

Google page speed scores are plain numbers, not a percentage. I think we could indicate this by changing the string to `%2$s/100` or something.

For the comment instead of `%2$s: speed score percentage (e.g. '95%').` maybe `%2$s: page speed score (e.g. '95%').`

Reply:
Got it! Thanks for your input.